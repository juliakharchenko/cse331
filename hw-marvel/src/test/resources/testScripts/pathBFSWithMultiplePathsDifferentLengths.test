# Tests finding the shortest path between two nodes in a graph
# that have multiple paths of different lengths from the first node
# to the second node.

LoadGraph nba nbaPlayers.csv

# Finds shortest path out of all paths from MAGIC-JOHNSON to DWYANE-WADE
FindPath nba MAGIC-JOHNSON DWYANE-WADE

# Finds shortest path out of all paths from KAREEM-ABDUL-JABBAR to MICHAEL-JORDAN
FindPath nba KAREEM-ABDUL-JABBAR MICHAEL-JORDAN