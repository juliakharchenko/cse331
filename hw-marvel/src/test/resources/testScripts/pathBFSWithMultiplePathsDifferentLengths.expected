# Tests finding the shortest path between two nodes in a graph
# that have multiple paths of different lengths from the first node
# to the second node.

loaded graph nba

# Finds shortest path out of all paths from MAGIC-JOHNSON to DWYANE-WADE
path from MAGIC-JOHNSON to DWYANE-WADE:
MAGIC-JOHNSON to LEBRON-JAMES via LAKERS
LEBRON-JAMES to DWYANE-WADE via CAVALIERS

# Finds shortest path out of all paths from KAREEM-ABDUL-JABBAR to MICHAEL-JORDAN
path from KAREEM-ABDUL-JABBAR to MICHAEL-JORDAN:
KAREEM-ABDUL-JABBAR to LEBRON-JAMES via LAKERS
LEBRON-JAMES to DWYANE-WADE via CAVALIERS
DWYANE-WADE to MICHAEL-JORDAN via BULLS